sources/libs/brutal/font/font.c

```diff
@@ -0,0 +1,74 @@
+#include <brutal/font/builtin.h>
+#include <brutal/font/font.h>
+
+float bfont_advance(BFont *font, Rune rune)
+{
+    switch (font->type)
+    {
+    case BFONT_BUILTIN:
```

Comment:
why not a function pointer in BFont* ? instead of a switch ?

Reply:
I felt like using switch cases that day, not that important the code is just a skeleton for the future TrueType renderer